test/k8s-integration/driver-config.go

```diff
@@ -127,10 +127,12 @@ func generateDriverConfigFile(testParams *testParameters) (string, error) {
 		snapshotClassName = "no-volumesnapshotclass"
 	}
 
-	caps = append(caps, "pvcDataSource")
+	if strings.Contains(testParams.storageClassFile, "sc-standard") {
```

Comment:
I think we want to check pvcDataSource on everything but extreme, not just on standard.

 (if we picked one we'd do balanced anyway)

Reply:
I was thinking hyperdisk also does not support cloning?

Reply:
We'll cross that bridge when we get to it. I think some SKUs will.